Yoast/yoast-components
# Turn snippet preview title red when the length score is lower than 5

## Summary

This PR can be summarized in the following changelog entry:

* Change the text color of the title in the snippet to red when the title is too long.

## Relevant technical choices:

* Rename descriptionLengthAssessment to descriptionLengthProgress because the object shows progress, not an assessment.
* Rename DraftJS to Draft.js

## Test instructions

This PR can be tested by following these steps:

* See https://github.com/Yoast/wordpress-seo/pull/9646

Connect https://github.com/Yoast/wordpress-seo/issues/9588
